site/posts/jupyter-notebooks-faq.ipynb

```diff
@@ -0,0 +1,173 @@
+    "\n",
+    "**A.** This is only a warning, not an error message. No fix is necessary. \n",
+    "\n",
+    "-----"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "**Q.** How do you specify what browser Jupyter opens with?\n",
+    "\n",
+    "**A.** If you don’t want to use your default browser (which Jupyter will automatically open), you can specify a different one with `jupyter lab —browser=chrome` (e.g., for Chrome)\n",
+    "\n",
+    "-----"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "**Q.** Jupyter opens to my base environment, not the environment I launched `jupyter lab` from. Is this an error?\n",
+    "\n",
+    "**A.** This is the expected behavior. You can double check your environment by typing `conda env list` from your Jupyter terminal to see all of your environments. The one with the asterisks next to it is your activated environment.\n",
+    "\n",
+    "-----"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "**Q.** The Jupyter workspace autocompletion of quotation marks and parenthesis is causing errors. Is there a way to disable this?\n",
+    "\n",
+    "**A.** Yes according to this this [stack overflow post](https://stackoverflow.com/questions/44216326/how-to-disable-auto-quotes-and-auto-brackets-in-jupyter-5-0#:~:text=2%20Answers&text=After%20executing%20the%20Python%20command,auto%2Dclosing%20quotes%20and%20brackets), in any code cell type: \n",
+    "\n",
+    "```\n",
+    "from notebook.services.config import ConfigManager\n",
+    "c = ConfigManager()\n",
+    "c.update('notebook', {\"CodeCell\": {\"cm_config\": {\"autoCloseBrackets\": False}}})\n",
+    "```\n",
+    "\n",
+    "You may have to restart the Jupyter session for the changes to take effect.\n",
+    "\n",
+    "-----"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "**Q.** Is converting a cell from code to raw a good way to basically “comment out” an entire cell?  Is that good practice or bad practice?\n",
+    "\n",
+    "**A.** This is the easiest way to \"comment out\" a cell.\n",
+    "\n",
+    "-----"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "**Q.** How do I use Jupyter with an R kernel (instead of Python)?\n",
+    "\n",
+    "**A.** You would need to create a new Conda environment with R and `jupyter` installed.\n",
+    "\n",
+    "-----"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "**Q.** Does Jupyter Lab care about being shut down gracefully?  If I you just `ctrl-C` it from the command-line, does that do anything bad?\n",
+    "\n",
+    "**A.** As long as your notebook is saved, `ctrl-C` is fine. Otherwise, you will lose the contents of your notebooks that weren’t saved before the shutdown.\n",
+    "\n",
+    "-----"
```

Comment:
Depending how users shut things down, it could still be running in the background.  Should we include something on `jupyter notebook list`?

Reply:
Good point Sheri! I addressed this in #139